Dump/plot_otg_move.py

```diff
@@ -115,6 +115,7 @@ def find_existing_plots():
     """
 
     cmd = 'ls ' + arc_dir + 'OTG_Plots/*.png > ' + zspace
+    cmd = cmd + '; ls ' + arc_dir + 'OTG_Plots/Ind_Plots/*.png > ' + zspace
```

Comment:
Use
cmd = f"ls {arc_dir}/OTG_Plots/*.png > {zspace}"

Why do you need to call ls twice?

Reply:
I've been classifying this type of fix as part of my efforts to bring the script to an MTA coding standard by using the glob package instead. So this issue wouldn't be covered in this PR, but is addressed later on.